javascript/src/main/components/Courses/CourseTable.js

```diff
@@ -21,24 +21,51 @@ export default ({courses,admin,deleteCourse}) => {
         )
     }
 
+    const sortCaret = (order, column) => {
+        const ascendingON = String.fromCharCode(0x25b2);
+        const descendingON = String.fromCharCode(0x25bc);
+        const ascendingOFF = String.fromCharCode(0x25b3);
+        const descendingOFF = String.fromCharCode(0x25bd);
+
+        if (!order) 
+            return (<span data-testid="sort">&nbsp;&nbsp;{descendingOFF}{ascendingOFF}</span>);
+        else if (order === 'asc') 
+            return (<span data-testid="sort-asc">&nbsp;&nbsp;{descendingOFF}<font color="red">{ascendingON}</font></span>);
+        else 
+            return (<span data-testid="sort-desc">&nbsp;&nbsp;<font color="red">{descendingON}</font>{ascendingOFF}</span>);
```

Comment:
I'd remove the non-breaking spaces:
```suggestion
        if (!order) 
            return (<span data-testid="sort">{descendingOFF}{ascendingOFF}</span>);
        else if (order === 'asc') 
            return (<span data-testid="sort-asc">{descendingOFF}<font color="red">{ascendingON}</font></span>);
        else 
            return (<span data-testid="sort-desc"><font color="red">{descendingON}</font>{ascendingOFF}</span>);
```